neard/src/shard_tracker.rs

```diff
@@ -148,6 +148,10 @@ impl ShardTracker {
         Ok(())
     }
 
+    pub fn get_current_epoch_id(&self) -> &EpochId {
+        &self.current_epoch_id
```

Comment:
This `current_epoch_id` should not be used externally. It is not guaranteed to be up-to-date.

Reply:
++